pkg/extensions/tbtc/events_log.go

```diff
@@ -0,0 +1,49 @@
+package tbtc
+
+import (
+	"sync"
+)
+
+type tbtcEventsLog struct {
```

Comment:
We have a similar structure in `pkg/client/events_tracking.go` as well as in `keep-core`. Even though I like `eventsLog` name better, I think we should stick to `*Track`/`*_tracking` naming for structures and file for consistency. I think this structure is going to provide the same features as the one in `events_tracking.go` except that it also offers to get all events.

Reply:
I got rid of the events log. It is not needed.